package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "moleculer-healthcheck-mongo-middleware",
+  "version": "0.0.0",
+  "description": "Wrapper of moleculer-healthcheck-middleware to check liveness of Mongo",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "npm install",
+    "test": "ava",
+    "pretest:debug": "npm run pretest",
+    "test:debug": "ava debug --host 0.0.0.0 --port 9229"
```

Comment:
Not sure but I think 9229 is the default port, so no need to overload the default.

Reply:
Fixed in 977666d4ab0c7c4fe11f67345642ad83c0411b51